cloud_research/README.md

```diff
@@ -0,0 +1,81 @@
+  * self managed (some of below and be counted as "hidden" costs):
+	* installation should be "easy"
+	* maintenance will require moderate effort (also depends on stability of underlying layers)
+		platform related:
+		* monitoring
+		* backups
+		* recovery plan - can be helpful also in case we will break something and it will take significant time to investigate and fix
+		openshift related:
+		(#TODO - get idea about daily mainatainance tasks and common issues)
+	* will we require test instance to prepare some acttions?
+
+* SSD vs. HDD: In our environment it should be enough to have HDD, with option to upgrade to SSD in case of need. Upgrade will consist of data migration to new volume, cloud platform should be well prepared for such scenario.
+
+* high probably, we will have to pay for RedHat linux subscription, is it possible to avoid that?
+  
+## Alternatives:
+  
+### Azure
+[cost calculator](https://azure.microsoft.com/en-us/pricing/calculator/)\
+B-tier vs. D-tier VM series - B is cheaper but less powerfull, D is "enterprise level"\
+[disk performance comparisson](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-types)\
+
+* OpenShift4 (managed):
+	* [cost (3 year commitment)](https://azure.com/e/59ac45f6d41b4e3eba11846817fcd11e)
+		* Upfront cost: ~$43,670.18
+		* Monthly cost: ~$2.90
+
+* OpenShift4 (self managed) :
+	*  [pay-as-you-go - installer default requirements](https://azure.com/e/2168c57a7dd144e0ab7d5e4f40c794ba)
+		* Estimated monthly cost: $1,841.09
+	*  [3 year commitment - installer default requirements](https://azure.com/e/19bb77f22082408fa73bfbeb85f1a1f2)
+		* Estimated monthly cost: $1,103.52
+	*  [pay-as-you-go - minimal requirements](https://azure.com/e/0172ab3e35bf4fbeaf68bc59d3542773)
+		* Estimated monthly cost: $1,004.51
+	*  [3 year commitment - minimal requirements](https://azure.com/e/8baba032d43d4433977298c04974d33c)
+		* Estimated monthly cost: $611.30
+
+  
+### AWS fedora + kubernetes:
+* for free
+* requires hands-on to be able to determine if our ansible based deployment is compatible with kubernetes and if is worth to migrate 
+	
+### PSI:
+Is running on OpenStack and is unstable - will cause lot of issue
+
+* **PSI + OpenShift3 (4?) (PHX manged)** 
+  * should be available in 3-4 months
+
+* **PSI + OpenShift4 (PHX self manged)**
+  * full time job - we expect lot of issues on multiple layers (hw, openstack, opneshit)
+  * first hw should be available 24. 5. 2020 and next bigger bunch 3 weeks later, PSI expects high demand for public clouds, but they should be able to onboard, still it will be our choice we should get in touch with them ASAP
+  
+* **PSI + Openshift (managed)**
+  * no public access currently, there is ongoing dicusssion about publict access, but is in early stage and can took 6-12 months (maybe longer).  
+
+
+## TODO
+* AWS research
+* links
+* try to get as close cost estimations as possible
+* research usefull services (alternative to redis, postgres, ?)
```

Comment:
Why is this important atm? I have the feeling that this could be left out for now.

Reply:
it was mentioned during discussion, definitely not priority. will create specific section for such items